src/main/java/org/concord/sensor/DeviceFinder.java

```diff
@@ -80,6 +80,9 @@ public static int getSensorDeviceID(short vendorId, short productId) throws Unkn
 			case 0x000B: // LabQuest 2
 				logger.fine("Found a Vernier LabQuest 2!");
 				return DeviceID.VERNIER_LAB_QUEST;
+			case 0x0010: // Go Direct
+				logger.fine("Found a Vernier Go Direct!");

```

Comment:
When Vernier refers to these sensors do they refer to them as "Go Direct!" or "GoDirect!"? I thought it was the latter, but we should follow their lead.

Reply:
On their website they use "Go Direct" with a space.